### PR TITLE
Update copyright information to 2016-2017.

### PR DESCRIPTION
**Pull request by kasiastreich**
Issue: https://github.com/corda/corda/issues/490

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 # General information about the project.
 project = u'R3 Corda'
-copyright = u'2016, R3 Limited'
```
Comment (chrisr3) [inline docs/source/conf.py]: I've done this already in my `chrisr3-documentation-fixes` branch. And Mike said we just want 2017.

Comment (gendal) [inline docs/source/conf.py]: didn't we start work on Corda in 2015?

Comment (chrisr3) [inline docs/source/conf.py]: 3:58 PM Chris Rankin
> "Documentation Copyright: copyright = u'2016, R3 Limited'"
Update to "2016-2017"?
Or just 2017?


4:33 PM Mike Hearn
> @chrisr 2017

Comment (gendal) [inline docs/source/conf.py]: i've emailed one of our lawyers for guidance on which we need (current year or 2015-2017). will comment back when she gets back to me

Comment (gendal) [inline docs/source/conf.py]: "Hi Richard -- While there are arguments both ways, general practice is to include just the most recent year, so “Copyright 2017 R3 Limited” is sufficient."

**Comment by kasiastreich** [comment]
Closing this PR, as @chrisr3 already solved it in his branch.

**Pull request closed without merging**